src/pkg/cli/compose/convert_test.go

```diff
@@ -8,7 +8,6 @@ import (
 	"github.com/DefangLabs/defang/src/pkg/term"
 	defangv1 "github.com/DefangLabs/defang/src/protos/io/defang/v1"
 	"github.com/compose-spec/compose-go/v2/types"
-	composeTypes "github.com/compose-spec/compose-go/v2/types"
```

Comment:
I suggest we always use `composeTypes` for this package, even when not strictly necessary.